### PR TITLE
IP - High Quality Surface Mesh Generation from Registered Range Data

### PR DESCRIPTION
**Pull request by tlow92**
Related changes for surface reconstruction.
Main development was done in cartographer base package. 
PR for that can be found here: https://github.com/tu-darmstadt-ros-pkg/cartographer/pull/1

### CODE REVIEW

File: cartographer_ros/CMakeLists.txt [hunk 1]
```diff
@@ -18,7 +18,6 @@ project(cartographer_ros)
 
 set(PACKAGE_DEPENDENCIES
   cartographer_ros_msgs
-  eigen_conversions
```
Comment (kdaun) [inline cartographer_ros/CMakeLists.txt:21]: Why did you remove the dependency? Is it not needed?

File: cartographer_ros/urdf/drz_telemax_complete.urdf [hunk 2]
```diff
@@ -0,0 +1,1653 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (kdaun) [inline cartographer_ros/urdf/drz_telemax_complete.urdf]: Please do not add files with proprietary robot information to public repositories.

**Review by kdaun — CHANGES_REQUESTED** [review]
Please remove the URDF file and check the dependency.

File: cartographer_ros/CMakeLists.txt (hunk 1 above)
Comment (tlow92) [reply cartographer_ros/CMakeLists.txt:21]: This change is coming from the base master. My branch is 2 commits ahead of this one, because I branched of master.
https://github.com/cartographer-project/cartographer_ros/pull/1278

File: cartographer_ros/urdf/drz_telemax_complete.urdf (hunk 2 above)
Comment (tlow92) [reply cartographer_ros/urdf/drz_telemax_complete.urdf]: :+1: Done. Was not yet finished with cleaning up.
